cobc/pplex.l

```diff
@@ -2337,6 +2466,7 @@ check_listing (const char *text, const unsigned int comment)
 		fprintf (cb_listing_file, "%6d%c", ++listing_line, c);
 	}
 
+	/* FIXME: what's the condition here? */
 	if (requires_listing_line && cb_source_format != CB_FORMAT_FREE &&
 	    strlen (text) > 6) {
```

Comment:
That's `cb_indicator_column - 1` and a check for "is there something to print afterwards (thinking about it: if the length is too small it should point to `""` instead of sequence area, that was buggy before)